setup.py

```diff
@@ -1,111 +1,32 @@
 #!/usr/bin/env python
+import os
+import subprocess
 
-#from distutils.core import setup
-#from distutils.extension import Extension
+import numpy
 from setuptools import setup
 from setuptools.extension import Extension
-from setuptools.config import read_configuration
-
-#import ah_bootstrap
-#import builtins
-
-import subprocess
-
-import os
-
-try:
-    import numpy 
-    include_dirs = [numpy.get_include()]
-except ImportError:
-    include_dirs = []
-    
-try:
-    from Cython.Build import cythonize
-    USE_CYTHON = True
-except ImportError:
-    USE_CYTHON = False
-
-if not os.path.exists('grizli/utils_c/interp.pyx'):
-    USE_CYTHON = False
-    
-if USE_CYTHON:
-    cext = '.pyx'
-else:
-    cext = '.c'
-
-print('C extension: {0}'.format(cext))
 
+include_dirs = [numpy.get_include()]
 if os.name == 'nt':
     # Windows
-    extensions = [
-        Extension("grizli.utils_c.interp", ["grizli/utils_c/interp"+cext],
-            include_dirs = include_dirs),
-        
-        Extension("grizli.utils_c.disperse", ["grizli/utils_c/disperse"+cext],
-            include_dirs = include_dirs),
-    ]
```

Comment:
What happens on Windows installation then?

Reply:
Great question.  We are not testing on Windows, so who knows.

I suppose we could add a Github actions runner on Windows and see what happens?  I suspect there's a ton of bugs that would break anything on Windows because of code like:

```python
path = ROOT + f"Prep/{dataset_id}"
```